InformationScripting/scripts/sink.py

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
Less implementation overhead is what I meant.

For example when testing: addTags ...| sink | tag -n=foo